lesson07/assignment/hello-world/src/App.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div id="app">
+    <header>
+      <div class="header-container">
+        <img alt="Coffee Project Logo" src="images/logo.png">
+        <h1>Coffee Project</h1>
+      </div>
+      <nav>
+        <ul>
+          <li><router-link to="/src/components/HomePage">Home</router-link></li>
```

Comment:
The router links should point to the route/url, not the path to the component.

`<li><router-link to="/">Home</router-link></li>
          <li><router-link to="/history">Images</router-link></li>
          <li><router-link to="/comparison">Accordion</router-link></li>
          <li><router-link to="/drinks">Grid System</router-link></li>`